app/gui/src/dashboard/layouts/Drive/Categories/categoriesHooks.tsx

```diff
@@ -22,6 +21,8 @@ import { newDirectoryId } from '#/services/LocalBackend'
 import { organizationIdToDirectoryId } from '#/services/RemoteBackend'
 import { getFileName } from '#/utilities/fileInfo'
 import LocalStorage from '#/utilities/LocalStorage'
+import { pickBackend } from '$/providers/backends'
```

Comment:
Try to remove `pickBackend` usages and simplify its logic, because it is redundant with `backendByCategory`.